src/Microsoft.DotNet.Darc/Darc/Operations/GetAssetOperation.cs

```diff
@@ -49,29 +55,37 @@ public override async Task<int> ExecuteAsync()
             List<Asset> matchingAssets =
                 (await remote.GetAssetsAsync(name: _options.Name, version: _options.Version, buildId: _options.Build)).ToList();
 
-            var queryDescription = new StringBuilder($"name '{_options.Name}'");
+            var queryDescription = new StringBuilder();
+
+            if (!string.IsNullOrEmpty(_options.Name))
+            {
+                queryDescription.Append($" named '{_options.Name}'");
+            }
+
+            if (_options.Build.HasValue)
+            {
+                queryDescription.Append($" from build '{_options.Build}'");
+            }
 
             if (!string.IsNullOrEmpty(_options.Version))
             {
-                queryDescription.Append($" and version '{_options.Version}'");
+                queryDescription.Append($" with version '{_options.Version}'");
             }
 
             if (targetChannel != null)
             {
                 queryDescription.Append($" on channel '{targetChannel.Name}'");
             }
 
-            if (_options.Build != null)
+            if (!_options.Build.HasValue)
```

Comment:
This should be options.maxAge

Reply:
I don't think so - we want to remove the age filter when we pass in the build

Reply:
Ahh right, and the MaxAgeInDays has a default has a default value, so we don't have to check it